ruby-bandwidth-iris.gemspec

```diff
@@ -19,7 +19,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency "faraday_middleware"
   spec.add_dependency "nori"
   spec.add_dependency "activesupport",">= 4.2.7"
-  spec.add_dependency "certified"
```

Comment:
Gotta tell you, this one took me hours to finally track down in our app. https://stackoverflow.com/a/75692478/4681454